portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase

```diff
@@ -125,14 +125,45 @@ definition {
 	}
 
 	@description = "Verify that a Date Field can be previewed"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test DateFieldCanBePreviewed {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133787 DateFieldCanBePreviewed pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Date");
+
+		FormFields.editHelpText(
+			fieldHelpText = "Test Help Text",
+			fieldName = "tip");
+
+		FormsAdminNavigator.gotoPreviewForm();
+
+		Navigator.selectSpecificWindow(windowTitle = "Form - Forms");
+
+		FormFields.viewDateField(
+			fieldLabel = "Date",
+			fieldName = "Date");
+
+		FormFields.viewFieldHelpText(
+			fieldHelpText = "Test Help Text",
+			fieldName = "Test Help Text");
 
+		FormFields.editDateCalendar(
+			fieldName = "Date",
+			fieldValue = "01/01/2019",
+			key_valueMonth = "January",
+			key_valueYear = "2019",
+			valueDay = "1");
```

Comment:
@lima-rafa, please just add the `key_` inside the macro, I mean, use the pattern:
```
key_valueMonth = "${valueMonth}",
```
Instead of:
```
key_valueMonth = "${key_valueMonth}",
```
In addition, is there any special reason for you set the year to 2019 instead of 2001?